deploy/docs/opentelemetry_collector.md

```diff
@@ -433,3 +436,31 @@ kubectl delete pvc --namespace ${NAMESPACE} --selector app=${RELEASE_NAME}-sumol
 ```
 
 [yq]: https://mikefarah.gitbook.io/yq/v/v3.x/
+
+## Traces
+### Load balancing using the gateway
+
+Open Telemetry supports [Trace ID aware load balancing](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/loadbalancingexporter/README.md).
+An example use case for load balancing is scaling a [cascading_filter](https://help.sumologic.com/Traces/03Advanced_Configuration/What_if_I_don't_want_to_send_all_the_tracing_data_to_Sumo_Logic%3F) that requires spans with same Trace ID to be send to the same collector instance.
+
+Sumologic kubernetes collection supports three layer architecture - with an agent, gateway and a collector - in order to perform Trace ID aware load balancing.
+Agent, if load balancing is configured, sends traces to the gateway. Gateway is configured with a load balancing exporter pointing to the collector headless service. Gateway may also be exposed outside cluster, allowing to load balance traces originating from outside kubernetes cluster.
+
+Sample config:
+
+```yaml
+sumologic:
+  traces:
+    enabled: true
+    loadBalancing: true
+  metrics:
+    enabled: false
```

Comment:
This is the default. No need to mention it

Reply:
```
  ### Metrics configuration
  ## Set the enabled flag to false for disabling metrics ingestion altogether.
  metrics:
    enabled: true
```

Reply:
Regarding misconfiguration, if loadBalancing is true but gateway is dissabled, it will be problematic. I'll change tge condition to check for both values

Reply:
Added missing lines. Thanks for catching this.